server/infrastructure/portal/mapper/PortalDataMapper.ts

```
@@ -0,0 +1,150 @@
+      department: {
+        code: student.dpmjCd,
+        name: student.dpmjNm,
+      },
+      major: {
+        code: student.mjorCd,
+        name: student.mjorNm,
+      },
+      secondaryMajor: student.the2MjorCd
+        ? {
+            code: student.the2MjorCd,
+            name: student.the2MjorNm!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)